app/components/ListClaimConditions/ListClaimConditions.tsx

```
@@ -0,0 +1,286 @@
+          <div className="mb-6">
+            <p className="mb-0 text-lg text-slate-400">
+              Claims for token ID: #{props.nft.id.toString()}
+            </p>
+            <p className="text-sm text-slate-300">
+              Any wallet can claim this token
+            </p>
+          </div>
+
+          {claimConditions.map((cc, i) => (
+            <div
+              key={i + '-' + cc.startTimestamp.toString()}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)